This is a minimal issue reproduction example

In the following tests, there are two async test cases which are run.  When running them with internet connection, all is fine.
However, when running them the first test fails but jest reports that it is the second one which failed.